recipes/ecell4_base/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "ecell4_base" %}
+{% set version = "2.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ecell/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 6342531d37f5e239a037c2cee13a580752b77b10f72858bd870a641a675aa3c3
+    patches:
+      - cmake-generator-vs2017.patch
+  - url: https://github.com/pybind/pybind11/archive/v2.4.3.tar.gz
+    sha256: 1eed57bc6863190e35637290f97a20c81cfe4d9090ac0a24f3bbf08f265eb71d
+    folder: pybind11
```

Comment:
Why are you grabbing the source for pybind11?

Reply:
Thanks for your review. I will try it after checking @isuruf 's review change.

Reply:
Because our project `ecell4_base` originally contains `pybind11` as a submodule and uses it.
On the other hand, we have requirements for `pybind11` too, but I think it is unnecessary.

Reply:
I don't think that this is OK.  You should be able to use an external version of pybind11.  Are you using some unreleased SHA of pybind11

Reply:
No, we use the released version of pybind11.
And I have come to think that it is not good to grab the source for pybind11 as you said.
I'll try to use an external version of pybind11.
Thanks.